usecases/monitoring/prometheus.go

```diff
@@ -743,6 +756,55 @@ func newPrometheusMetrics() *PrometheusMetrics {
 			Help:    "Number of tokens processed per request",
 			Buckets: []float64{1, 10, 50, 100, 500, 1000, 10000, 100000, 1000000},
 		}, []string{"tokenizer"}),
+		OpenAIRequests: promauto.NewCounterVec(prometheus.CounterOpts{
+			Name: "openai_requests_total",
```

Comment:
The new metrics we add, we try to keep it consistent with `weaviate_` naming standard. 

examples: [http, grpc servers,](https://github.com/weaviate/weaviate/blob/stable%2Fv1.30/adapters/handlers/rest/configure_api.go#L218-L219) [config manager](https://github.com/weaviate/weaviate/blob/stable%2Fv1.30/usecases/config/runtime/manager.go#L70-L91), [schema metrics](https://github.com/weaviate/weaviate/blob/stable%2Fv1.30/cluster/schema/schema.go#L79-L90).

Rationale is we can have weaviate specific metrics with common prefix `weaviate_`. (also don't conflict with any internal `openai_*` metrics (or similiar in the future)